libs/auth/scripts/program_production_system_administrator_java_card_without_vx_admin.ts

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs/promises';
+import path from 'path';
+import { DirResult, dirSync } from 'tmp';
+import { extractErrorMessage } from '@votingworks/basics';
+
+import { PROD_VX_CERT_AUTHORITY_CERT_PATH } from '../src';
+import { CERT_EXPIRY_IN_DAYS, constructMachineCertSubject } from '../src/certs';
+import { createCert } from '../src/cryptography';
+import { getRequiredEnvVar } from '../src/env_vars';
+import { JavaCard } from '../src/java_card';
+import {
+  generatePrivateKey,
+  programSystemAdministratorJavaCard,
+} from './utils';
+
+const jurisdiction = getRequiredEnvVar('VX_MACHINE_JURISDICTION');
+const vxPrivateKeyPath = getRequiredEnvVar('VX_PRIVATE_KEY_PATH');
+
+async function instantiateJavaCardWithOneOffVxAdminPrivateKeyAndCertAuthorityCert(
+  tempDirectoryPath: string
+): Promise<JavaCard> {
+  const vxAdminPrivateKey = await generatePrivateKey();
+  const vxAdminPrivateKeyPath = path.join(
+    tempDirectoryPath,
+    'vx-admin-private-key.pem'
+  );
```

Comment:
can this be explicitly deleted rather than rely on tmp cleanup later?

Reply:
Ah yup, so we are explicitly removing, in the `finally` block